auto3dseg/algorithm_templates/segresnet/scripts/dummy_runner.py

```diff
@@ -0,0 +1,180 @@
+
+        with open(data_stats_file) as f_data_stat:
+            data_stat = yaml.full_load(f_data_stat)
+
+        self.max_shape = [0, 0, 0]
+        if parser.get_parsed_content("resample"):
+            pixdim = parser.get_parsed_content("resample_resolution")
+
+            for _k in range(len(data_stat["stats_by_cases"])):
+                image_shape = data_stat["stats_by_cases"][_k]["image_stats"]["shape"]
+                image_shape = np.squeeze(image_shape)
+                image_spacing = data_stat["stats_by_cases"][_k]["image_stats"][
+                    "spacing"
+                ]
+                image_spacing = np.squeeze(image_spacing)
+                image_spacing = [np.abs(image_spacing[_i]) for _i in range(3)]
+
+                new_shape = [
+                    int(
+                        np.ceil(float(image_shape[_l]) * image_spacing[_l] / pixdim[_l])
+                    )
+                    for _l in range(3)
+                ]
+                if np.prod(new_shape) > np.prod(self.max_shape):
+                    self.max_shape = new_shape
+        else:
+            for _k in range(len(data_stat["stats_by_cases"])):
+                image_shape = data_stat["stats_by_cases"][_k]["image_stats"]["shape"]
+                image_shape = np.squeeze(image_shape)
+
+                if np.prod(image_shape) > np.prod(self.max_shape):
+                    self.max_shape = image_shape
+        print("max_shape", self.max_shape)
+
+        self.sliding_inferrer = SlidingWindowInferer(
+            roi_size=self.roi_size,
+            sw_batch_size=1,
+            overlap=0.625,
+            mode="gaussian",
+            cache_roi_weight_map=True,
+            progress=False,
+            cpu_thresh=512 ** 3 // output_classes,
+        )
+
+    def run(
+        self,
+        num_images_per_batch,
+    ):
+        scaler = GradScaler()
+
+        num_epochs = 2
+        num_iterations = 6
+        num_iterations_validation = 1
+
+        print("num_images_per_batch", num_images_per_batch)
+
+        for _i in range(num_epochs):
+            # training
+            print("------  training  ------")
+
+            self.model.train()
+
+            for _j in range(num_iterations):
+                print("iteration", _j + 1)
+
+                inputs = torch.rand(
+                    (
+                        num_images_per_batch,
+                        self.input_channels,
+                        self.roi_size[0],
+                        self.roi_size[1],
+                        self.roi_size[2],
+                    )
+                )
+                labels = torch.rand(
+                    (
+                        num_images_per_batch,
+                        self.label_channels,
+                        self.roi_size[0],
+                        self.roi_size[1],
+                        self.roi_size[2],
+                    )
+                )
```

Comment:
since the dummy label is not an index map (0,1,2,3,..) it won't be correctly one_hot encoded during loss calculation